src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -502,6 +502,18 @@ private function isInteractiveInput($inputStream): bool
      */
     private function readInput($inputStream, Question $question): string|false
     {
+        if (null !== $question->getTimeout() && $this->isInteractiveInput($inputStream)) {
+            $read = [$inputStream];
+            $write = null;
+            $except = null;
+            $timeoutSeconds = $question->getTimeout();
+            $changedStreams = stream_select($read, $write, $except, $timeoutSeconds);
+
+            if (0 === $changedStreams) {
```

Comment:
```suggestion
            if (0 === stream_select([$inputStream], null, null, $question->getTimeout())) {
```

Reply:
You cannot do that, first 3 params are references, only variables can be passed by ref. We can inline only the stream_select call and the timeout.